### ISSUE TITLE
Barcode check using more RAM than allocated

### ISSUE THREAD

**Issue by TomHarrop** [TomHarrop/tcdemux#7]
Need to allow for overhead when setting `Xmx` in demuxbyname.

Use 0.8 or so of allocated?

**Comment by TomHarrop** [comment]
```python
    params:
        mem_gb = lambda wildcards, resources:
            int(resources.mem_gb * 0.5)
```

**Comment by TomHarrop** [comment]
You can also set the threads although it's not documented.

`--threads=2`

**Comment by TomHarrop** [comment]
Fixed in [0.0.10](https://github.com/TomHarrop/tcdemux/releases/tag/0.0.10)
